__tests__/app.test.js

```diff
@@ -37,12 +37,42 @@ describe('/api', () => {
     return request(app)
     .get('/api')
     .then((response) => {
-      console.log(response.body)
       expect(response.body.endpoints).toEqual(endpoints)
     })
   })
 })
 
+describe('/api/articles/:article_id', () => {
+  test('GET 200: Responds with an article when we query by id', () => {
+  return request(app)
+    .get("/api/articles/1")
+    .expect(200)
+    .then(({ body }) => {
+      const { article } = body;
+      console.log(article)
+      expect(article).toMatchObject({
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: "2020-07-09T20:11:00.000Z",
+        votes: 100,
+        article_img_url: "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+    });
+    });
+  })
+
+  test('GET 404: article not found for non-existing article_id', () => {
+    return request(app)
+        .get('/api/articles/123456')
+        .expect(404)
+        .then(({ body }) => {
+          const { msg } = body;
+            expect( msg ).toBe('we dont find yor article');
+        });
```

Comment:
This is good! There is another error test you should also be doing, what happens if the article_id is not a number ?
`/api/articles/banana`